src/plugins/console/public/lib/es/es.ts

```diff
@@ -24,44 +19,37 @@ export function getContentType(body: unknown) {
   return 'application/json';
 }
 
-export function send(
-  method: string,
-  path: string,
-  data: string | object,
-  { asSystemRequest }: SendOptions = {},
-  withProductOrigin: boolean = false
-) {
-  const wrappedDfd = $.Deferred();
+let _http: HttpSetup;
 
-  const options: JQuery.AjaxSettings = {
-    url:
-      '../api/console/proxy?' +
-      stringify({ path, method, ...(withProductOrigin && { withProductOrigin }) }, { sort: false }),
-    headers: {
-      'kbn-xsrf': 'kibana',
-      ...(asSystemRequest && { 'kbn-system-request': 'true' }),
-    },
-    data,
-    contentType: getContentType(data),
-    cache: false,
-    crossDomain: true,
-    type: 'POST',
-    dataType: 'text', // disable automatic guessing
-  };
+export const initHttp = (http: HttpSetup) => {
```

Comment:
can we avoid doing it? We are trying to avoid of setting services using that way. Please use more legal way for doing it